src/shared/card/VideoCard.js

```diff
@@ -0,0 +1,40 @@
+import "./VideoCard.css";
+import { Link } from "react-router-dom";
+import { useVideo } from "../../context/videosContext";
+const VideoCard = () => {
+  const { videos, loader } = useVideo();
+  return (
+    <div className="video-listing-card bg-theme-changer">
+      {loader && <div className="flex-center"> Loading the Videos </div>}
+      {videos.map((item) => {
+        return (
+          <div key={item._id}>
+            <Link to={`/player/${item._id}`}>
+              <img src={item.img} className="video-images" alt={item.title} />
```

Comment:
use object-fit and object-position properties, your thumbnails will look much better

Reply:
Okk